service/src/intTest/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentComponentTest.java

```diff
@@ -130,24 +137,28 @@ public void When_NewAccessDocumentTaskIsStartedAndThenUpdated_Expect_DatabaseAnd
     }
 
     @Test
+    @Disabled
```

Comment:
Same as above, is this supposed to be disabled?

Reply:
I'll add a TODO item to change that in the future